### PR TITLE
Updates exist cmdlet: Update-AzureSqlDatabaseLongTermRetentionBackup to achieve change LTR backup access tier goal.

### PR DESCRIPTION
**Pull request by lululilliancoding**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Sql/Sql/Database Backup/Cmdlet/UpdateAzureSqlDatabaseLongTermRetentionBackup.cs [hunk 1]
```diff
@@ -95,6 +101,11 @@ public class UpdateAzureSqlDatabaseLongTermRetentionBackup : AzureSqlDatabaseLon
             ValueFromPipelineByPropertyName = true,
             Position = 3,
             HelpMessage = "The name of the backup.")]
+        [Parameter(Mandatory = true,
+            ParameterSetName = UpdateBackupStorageAccessTierSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
```
Comment (NoriZC) [inline src/Sql/Sql/Database Backup/Cmdlet/UpdateAzureSqlDatabaseLongTermRetentionBackup.cs:107]: Is the `Position = 3` necessary here? We recommand to have less than 3 parameters with position, this is why the analysis gives an error.

Comment (lululilliancoding) [reply src/Sql/Sql/Database Backup/Cmdlet/UpdateAzureSqlDatabaseLongTermRetentionBackup.cs:107]: ACK I'll change it to Named

File: src/Sql/Sql/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Update exist cmdlets 'UpdateAzureSqlDatabaseLongTermRetentionBackup' for Azure SQL sterling DB to change long-term retention backup storage access tier
```
Comment (NoriZC) [inline src/Sql/Sql/ChangeLog.md:21]:
```suggestion
* Updated exist cmdlets 'UpdateAzureSqlDatabaseLongTermRetentionBackup' for Azure SQL sterling DB to change long-term retention backup storage access tier
```

**Review by NoriZC — CHANGES_REQUESTED** [review]

File: src/Sql/Sql/ChangeLog.md (hunk 2 above)
Comment (lululilliancoding) [reply src/Sql/Sql/ChangeLog.md]: fixed

**Review by NoriZC — APPROVED** [review]

**Review by NoriZC — CHANGES_REQUESTED** [review]
Another concern - Have you [regenerated the help files](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md)? And since you are adding a new parameter set, it is needed for you to [write and record a test](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/azure-powershell-developer-guide.md#adding-tests) for it. Please let me know if you are still confused on the steps after reading the links.

**Review by NoriZC — APPROVED** [review]

**Pull request merged**